includes/class-wc-payments-webhook-processing-service.php

```diff
@@ -161,6 +161,12 @@ public function process( array $event_body ) {
 			case 'charge.expired':
 				$this->process_webhook_expired_authorization( $event_body );
 				break;
+			case 'charge.succeeded':
```

Comment:
Are we sure that server forwarding those events?

Reply:
You are right, the server is not. I realized this yesterday and changed the events locally. I just pushed this change in 2ad8f36.